### PR TITLE
Github Actions to test Dockerfile 🐳

### PR DESCRIPTION
**Pull request by pedrofurtado**
@shirts I've added a additional test in CI, to check if our Dockerfile (used by dev purposes, in order to run local tests) is working well with repo file changes 👍

### CODE REVIEW

**Comment by pedrofurtado** [comment]
@shirts 🤝 🤓

**Comment by pedrofurtado** [comment]
@shirts Your review/eyes on it will be awesome🤝 🤓

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,6 @@
 FROM ruby:3.0.0-alpine
 RUN apk update && apk add git
-RUN mkdir -p /app/
 WORKDIR /app/
-VOLUME /app/
-COPY . /app/
+COPY . .
```
Comment (shirts) [inline Dockerfile:4]: What's this doing – copying all files into the same directory?

Comment (pedrofurtado) [reply Dockerfile:4]: Copying all files from host to `/app` inside container

**Pull request merged**